diff --git a/node_modules/@shopify/react-native-skia/android/cpp/api b/node_modules/@shopify/react-native-skia/android/cpp/api
index 1eabfba..a861dbf 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/api
+++ b/node_modules/@shopify/react-native-skia/android/cpp/api
@@ -1 +1 @@
-/private/var/folders/q0/2f1y6j813zs38cn008hf8jcr0000gn/T/tmp-1848Fjq1nC2rRpUM/node_modules/@shopify/react-native-skia/cpp/api
\ No newline at end of file
+/Users/mrousavy/Projects/react-native-vision-camera/example/node_modules/@shopify/react-native-skia/cpp/api
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/jsi b/node_modules/@shopify/react-native-skia/android/cpp/jsi
index 5df4067..856ac16 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/jsi
+++ b/node_modules/@shopify/react-native-skia/android/cpp/jsi
@@ -1 +1 @@
-/private/var/folders/q0/2f1y6j813zs38cn008hf8jcr0000gn/T/tmp-1848Fjq1nC2rRpUM/node_modules/@shopify/react-native-skia/cpp/jsi
\ No newline at end of file
+/Users/mrousavy/Projects/react-native-vision-camera/example/node_modules/@shopify/react-native-skia/cpp/jsi
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/rnskia b/node_modules/@shopify/react-native-skia/android/cpp/rnskia
index afe48cc..42c13be 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/rnskia
+++ b/node_modules/@shopify/react-native-skia/android/cpp/rnskia
@@ -1 +1 @@
-/private/var/folders/q0/2f1y6j813zs38cn008hf8jcr0000gn/T/tmp-1848Fjq1nC2rRpUM/node_modules/@shopify/react-native-skia/cpp/rnskia
\ No newline at end of file
+/Users/mrousavy/Projects/react-native-vision-camera/example/node_modules/@shopify/react-native-skia/cpp/rnskia
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/skia b/node_modules/@shopify/react-native-skia/android/cpp/skia
index b90a36c..66cfa51 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/skia
+++ b/node_modules/@shopify/react-native-skia/android/cpp/skia
@@ -1 +1 @@
-/private/var/folders/q0/2f1y6j813zs38cn008hf8jcr0000gn/T/tmp-1848Fjq1nC2rRpUM/node_modules/@shopify/react-native-skia/cpp/skia
\ No newline at end of file
+/Users/mrousavy/Projects/react-native-vision-camera/example/node_modules/@shopify/react-native-skia/cpp/skia
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/utils b/node_modules/@shopify/react-native-skia/android/cpp/utils
index b07b9b3..b957a0b 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/utils
+++ b/node_modules/@shopify/react-native-skia/android/cpp/utils
@@ -1 +1 @@
-/private/var/folders/q0/2f1y6j813zs38cn008hf8jcr0000gn/T/tmp-1848Fjq1nC2rRpUM/node_modules/@shopify/react-native-skia/cpp/utils
\ No newline at end of file
+/Users/mrousavy/Projects/react-native-vision-camera/example/node_modules/@shopify/react-native-skia/cpp/utils
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/react-native-skia.podspec b/node_modules/@shopify/react-native-skia/react-native-skia.podspec
index 2e4868f..5dc8c9b 100644
--- a/node_modules/@shopify/react-native-skia/react-native-skia.podspec
+++ b/node_modules/@shopify/react-native-skia/react-native-skia.podspec
@@ -23,7 +23,8 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.pod_target_xcconfig = {
     'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) SK_GL=1 SK_METAL=1',
-    'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17'
+    'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
+    'DEFINES_MODULE' => 'YES'
   }
 
   s.frameworks = 'GLKit', 'MetalKit'
